vernac/g_vernac.mlg

```diff
@@ -409,6 +413,13 @@ GRAMMAR EXTEND Gram
       | "{";fs = record_fields; "}" -> { RecordDecl (None,fs) }
       |  -> { Constructors [] } ] ]
   ;
+  record_definition:
+    [ [ oc = opt_coercion; id = ident_decl; indpar = binders;
```

Comment:
Just to be clear, note that this drops the `OPT [ "|" binders ]` included in `inductive_definition`.  This clause doesn't apply to records?

Reply:
Indeed, it doesn't: 

```
Record foo (n : nat) | (m : nat) := {}.
(* Error: Records do not support the "|" syntax. *)
```